vic/drivers/classic/src/get_global_param.c

```diff
@@ -732,83 +551,79 @@ get_global_param(FILE *gp)
-            else if (strcasecmp("ALMA_OUTPUT", optstr) == 0) {
+
+            /*************************************
+               Fail when deprecated options are used.
+            *************************************/
+            else if (strcasecmp("ARC_SOIL", optstr) == 0) {
                 sscanf(cmdstr, "%*s %s", flgstr);
                 if (strcasecmp("TRUE", flgstr) == 0) {
-                    options.ALMA_OUTPUT = true;
-                }
-                else {
-                    options.ALMA_OUTPUT = false;
+                    log_err("\"ARC_SOIL\" is no longer a supported option.  "
+                            "Please convert your soil parameter file and "
+                            "remove this option from your global file");
                 }
             }
-            else if (strcasecmp("MOISTFRACT", optstr) == 0) {
+            else if (strcasecmp("ARNO_PARAMS", optstr) == 0) {
                 sscanf(cmdstr, "%*s %s", flgstr);
                 if (strcasecmp("TRUE", flgstr) == 0) {
-                    options.MOISTFRACT = true;
+                    log_err(
+                        "Please change \"ARNO_PARAMS  TRUE\" to \"BASEFLOW  "
+                        "NIJSSEN2001\" in your global parameter file.");
                 }
                 else {
-                    options.MOISTFRACT = false;
+                    log_err(
+                        "Please change \"ARNO_PARAMS  FALSE\" to \"BASEFLOW  "
+                        "ARNO\" in your global parameter file.");
                 }
             }
-            else if (strcasecmp("PRT_HEADER", optstr) == 0) {
+            else if (strcasecmp("NIJSSEN2001_BASEFLOW", optstr) == 0) {
                 sscanf(cmdstr, "%*s %s", flgstr);
                 if (strcasecmp("TRUE", flgstr) == 0) {
-                    options.PRT_HEADER = true;
+                    log_err(
+                        "Please change \"NIJSSEN2001_BASEFLOW  TRUE\" to "
+                        "\"BASEFLOW  NIJSSEN2001\" in your global parameter "
+                        "file.");
                 }
                 else {
-                    options.PRT_HEADER = false;
+                    log_err(
+                        "Please change \"NIJSSEN2001_BASEFLOW  FALSE\" to "
+                        "\"BASEFLOW  ARNO\" in your global parameter file.");
                 }
             }
-            else if (strcasecmp("PRT_SNOW_BAND", optstr) == 0) {
-                sscanf(cmdstr, "%*s %s", flgstr);
-                if (strcasecmp("TRUE", flgstr) == 0) {
-                    options.PRT_SNOW_BAND = true;
-                }
-                else {
-                    options.PRT_SNOW_BAND = false;
-                }
+            else if (strcasecmp("VEGLIB_VEGCOVER", optstr) == 0) {
+                log_err("The option VEGLIB_VEGCOVER has been replaced by "
+                        "VEGLIB_FCAN.  Please edit your global parameter "
+                        "file and re-run.");
```

Comment:
So why are we keeping this informative error statement helping users convert from the old `VEGLIB_VEGCOVER` to the new `VEGLIB_FCAN`, when on lines 692-701 we're eliminating a similar message regarding `VEGCOVER_SRC`?  Seems to me we should be consistent.  I don't have an opinion as to which path we take, just that it's consistent.